README.md

```diff
@@ -15,6 +15,27 @@ Currently this tool supports only PostgreSQL but we aim to add support for other
 
 Requires Python 3.5 or newer.
 
+## Installation
+Get up and running quickly by using the steps below...
+
+**Prerequistes**
+
+Recommended to use **Fedora Linux Distrubution** or you might need to install additional packages/perform additional steps than those listed below
+- Python 3.5 or newer
+- Python Pip
+- Git
+- make
+- psycopg2-binary
```

Comment:
If pip cmd is already in the example, for the quick start could have also specified the command how to install it.
`pip install psycopg2-binary`
Actually binary is not required, psycopg2 is required, but whether to install it as binary or not, it's up to the user.